_config/products.yml

```diff
@@ -728,13 +745,14 @@ jbt_core:
             md5_url: http://sourceforge.net/projects/jboss/files/JBossTools/jbosstools4.3.0.x/jbosstools-4.3.0.Alpha1_2015-02-15_14-44-55-B13-browsersim-standalone.zip.MD5
           - name: Release Notes
             url: http://tools.jboss.org/documentation/whatsnew/index.html
-      4.3.0.Nightly:
-        update_site_url: http://download.jboss.org/jbosstools/updates/nightly/mars/
+      4.3.x.Nightly:
+        update_site_url: http://download.jboss.org/jbosstools/mars/snapshots/updates/
```

Comment:
In /mars/snapshots/updates/ we see these files are links to the latest generated discovery site composites, which includes the latest JBT + JBT Central + target platforms:

lrwxrwxrwx 1 10116 10116   47 Sep 10 22:02 compositeContent.xml -> discovery.central/4.3.mars/compositeContent.xml
lrwxrwxrwx 1 10116 10116   49 Sep 10 22:02 compositeArtifacts.xml -> discovery.central/4.3.mars/compositeArtifacts.xml

So... that's the last available discovery site build at the moment, so yes, it's fine. 

I've kicked a new build of the discovery site so you'll see snapshot content in there instead of the CR2b staging stuff, just in case there's something new for 4.3.1.CR1.

But of course since ALL the other jobs are currently disabled, you won't see anything new until someone re-enables the jobs. :D